go-controller/pkg/cluster/gateway_localnet.go

```diff
@@ -72,9 +73,16 @@ func delIptRules(ipt util.IPTablesHelper, rules []iptRule) {
 	}
 }
 
+const natFwMark string = "0x466"
+
 func generateGatewayNATRules(ifname string, clusterCIDRs []string) []iptRule {
 	// Allow packets to/from the gateway interface in case defaults deny
 	rules := make([]iptRule, 0)
+	rules = append(rules, iptRule{
+		table: "filter",
+		chain: "FORWARD",
+		args:  []string{"-m", "mark", "--mark", natFwMark, "-j", "ACCEPT"},
+	})
```

Comment:
Are rules at L86-96 valid anymore? Since, @L84 we are letting all the marked packets in the rules @L86-96 is a no-op?